docs/package.json

```diff
@@ -4,20 +4,24 @@
   "private": true,
   "scripts": {
     "docusaurus": "docusaurus",
-    "start": "yarn enrich-document-metadata && docusaurus start",
+    "start": "yarn glossary-parse && yarn glossary && yarn enrich-document-metadata && docusaurus start",
     "validate:inputs": "./validate-api.js publish '/' '../hydra-node/golden/ReasonablySized (ClientInput (Tx BabbageEra)).json'",
     "validate:outputs": "./validate-api.js subscribe '/' '../hydra-node/golden/ReasonablySized (ServerOutput (Tx BabbageEra)).json'",
     "validate": "yarn validate:inputs && yarn validate:outputs",
-    "build": "yarn enrich-document-metadata && docusaurus build",
+    "build": "yarn glossary-parse && yarn glossary && yarn enrich-document-metadata && docusaurus build",
```

Comment:
Seems like these commands are used always together, does it make sense to have a single `yarn glossary` which does `yarn docusaurus parse && yarn docusaurus glossary`?